src/components/ChoreStep.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Checkbox from '@mui/material/Checkbox';
+import IconButton from '@mui/material/IconButton';
+import CommentIcon from '@mui/icons-material/Comment';
+
+export default function CheckboxList(props) {
+  const [checked, setChecked] = React.useState([0]);
+  const {sampleSteps} = props
+
+  const handleToggle = (value) => () => {
+    const currentIndex = checked.indexOf(value);
```

Comment:
Instead of using an array of values here, you could also use a Set which would be easier to check/add/remove values

```
[checked, setChecked] = Reace.useState(new Set());

const handleToggle = (value) => {
  if(checked.has(value)) {
    checked.delete(value);
  } else {
    checked.add(value);
  }
}
```